scripts/memory_logger.py

```diff
@@ -813,7 +813,11 @@ def buildGraph(self):
 
             # Populate the graph
             for records in value_list:
-                tmp_memory.append(decimal.Decimal(records[1]) / self.multiples)
+                if decimal.Decimal(records[1]) == 0:
```

Comment:
O I C. If that loop keeps going (which it will when largest_memory == 0) the multiplier will also eventually become 0.

Updating PR...